packages/block-editor/src/components/colors-gradients/style.scss

```diff
@@ -37,4 +37,40 @@
 	.block-editor-block-inspector & .components-base-control {
 		margin-bottom: inherit;
 	}
+
+	.components-dropdown {
+		display: block;
+	}
+
+	// Allow horizontal overflow so the size-increasing color indicators don't cause a scrollbar.
+	.components-navigator-screen {
+		overflow-x: visible;
+	}
+
 }
+
+@include break-medium() {
+	.block-editor-panel-color-gradient-settings__dropdown .components-popover__content {
+		margin-right: #{ math.div($sidebar-width, 2) + $grid-unit-20 } !important;
+		margin-top: #{ -($grid-unit-60 + $grid-unit-15) } !important;
+	}
+}
+
+.block-editor-panel-color-gradient-settings__item {
+	// @todo: this can be removed when https://github.com/WordPress/gutenberg/pull/37028 lands.
+	.component-color-indicator {
```

Comment:
37028 has been merged, so these rules can be removed. That gets us to this point at least:
<img width="316" alt="Screenshot 2021-12-02 at 16 59 51" src="https://user-images.githubusercontent.com/1204802/144457539-8379c607-f851-4a67-8c83-37dfe39c2557.png">

The diagonal line is missing. I'm realizing that this diagonal is scoped to work only in global styles: https://github.com/WordPress/gutenberg/pull/36994/files#diff-448ace8cb198438b19f4fc13f3107589730cf0aea599c46fa935e87ff1b70816R54

Where's a good place to move that rule? The diagonal should be shown when a color is _unset_, and the transparent checkerboard can be shown when a color is explicitly transparent. Any thoughts?

Reply:
The vertical alignment of the circles also seems off in the above capture

Reply:
A fix for that was merged in https://github.com/WordPress/gutenberg/pull/37028, I wonder if a rebase would fix it?

Reply:
_(Ignore this comment in case we're deleting this rule)_

We should try to avoid using internal classnames for styling `@wordpress/components` (same as comment above)

Reply:
We need this rule because of the empty line but the class enhancement was done.